2019_03_12_InputOutput-Demo1/src/SumNumberFromFile.java

```diff
@@ -0,0 +1,29 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+public class SumNumberFromFile {
+	
+	public static void main(String[] args) {
+			
+		try (BufferedReader reader = new BufferedReader (new FileReader ("number.txt"))){
+			
+			Stream<String> lines = reader.lines();
+			int sum = lines
+				    .map(line -> line.split("\\n")).flatMap(Arrays::stream)
```

Comment:
This looks a bit like over complication: you do not need split("\\n") and flatMap(), you may immediately start parsing int.